cpu/stm32/periph/spi.c

```diff
@@ -256,6 +256,11 @@ static void _transfer_dma(spi_t bus, const void *out, void *in, size_t len)
     dma_wait(spi_config[bus].rx_dma);
     dma_wait(spi_config[bus].tx_dma);
 
+#ifdef CPU_LINE_STM32F334x8
```

Comment:
Are you sure this is only valid for stm32f334r8 ? @bergzand any opinion here ?

Reply:
This is mandatory for f334r8, not for f746zg, I don't know for others.

Reply:
Just tried with f303k8 and this seems to be also needed so I will generalize it to CPU_FAM_STM32F3 and move it to #14764.